src/python/pants/backend/project_info/peek.py

```diff
@@ -41,6 +49,9 @@ def register_options(cls, register):
             "--output",
             type=OutputOptions,
             default=OutputOptions.JSON,
+            removal_version="2.9.0.dev0",
+            removal_hint="Output will always be JSON. If you need the raw BUILD file contents, "
+            "look at it directly!",
```

Comment:
It would be better for this to be in a dedicated change so that it shows up in the Changelog under User API Changes